mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -274,6 +274,156 @@ func TestAccComputeSecurityPolicy_withRateLimitWithRedirectOptions(t *testing.T)
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withRecaptchaOptionsConfig(project, spName, keyDisplayNameUpdate),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withEmptyRedirectSiteKey(spName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccComputeSecurityPolicy_withHeadAction(t *testing.T) {
+	t.Parallel()
+
+	spName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	headerName := fmt.Sprintf("tf-test-header-name-%s", randString(t, 10))
+	headerNameUpdate := fmt.Sprintf("tf-test-header-name-update-%s", randString(t, 10))
+	headerValue := fmt.Sprintf("tf-test-header-value-%s", randString(t, 10))
+	headerValueUpdate := fmt.Sprintf("tf-test-header-value-update-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeSecurityPolicyDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeSecurityPolicy_withoutHeadAction(spName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withHeadAction(spName, headerName, headerValue),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withHeadAction(spName, headerNameUpdate, headerValueUpdate),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withMultipleHeaders(spName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withoutHeadAction(spName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+func testAccComputeSecurityPolicy_withRecaptchaOptionsConfig(project, spName, keyDisplayName string) string {
+	return fmt.Sprintf(`
+resource "google_recaptcha_enterprise_key" "primary" {
+	display_name = "%s"
+
+	labels = {
+		label-one = "value-one"
+	}
+
+	project = "%s"
+
+	web_settings {
+		integration_type              = "INVISIBLE"
+		allow_all_domains             = true
+		allowed_domains               = ["localhost"]
+	}
+}
+
+resource "google_compute_security_policy" "policy" {
+  name        = "%s"
```

Comment:
Nit: It looks like there's a mix of <tab> and <space> indentations here. Perhaps worth changing them to be consistent (I believe I've seen <space> more commonly used in Terraform strings).

Same nit for the example in the docs.

Reply:
Thanks for the comment. Made the change.